src/s3fs_util.cpp

```diff
@@ -1340,6 +1340,17 @@ void show_help ()
     "        Please use this option when the directory in the bucket is\n"
     "        only \"dir/\" object.\n"
     "\n"
+    "   use_wtf8 - support arbitrary file system encoding.\n"
+    "        S3 requires all object names to be valid utf-8. But some\n"
+    "        clients, notably Windows NFS clients, use their own encoding.\n"
+    "        This option re-encodes invalid utf-8 object names into valid\n"
+    "        utf-8 by mapping offending codes into a 'private' codepage of the\n"
+    "        Unicode set.\n"
+    "        Useful on clients not using utf-8 as their file system encoding.\n"
+    "\n"
+    "   notsup_compat_dir (not support compatibility directory types)\n"
```

Comment:
Please remove L:1351 to L:1353 about "notsup_compat_dir", because it is already explained above this line.

Reply:
Right, I  left it in by mistake.  Fixed.

Reply:
Thanks.